transform/surveys/023.0112.json

```diff
@@ -31,10 +104,30 @@
       ]
     },
     {
-      "title": "3. Retail Turnover",
+      "title": "3. Commodoties - Retail Turnover",
       "meta": "(£)",
       "questions": [
         {
+          "text": "Food",
```

Comment:
I don't think we need to add these questions. Looking at form type 0112 on https://preprod-surveys-launch.eq.ons.digital/ it only has Total Retail Sales.

So you can remove questions 22, 23, 24, 25 and 26